app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -22,7 +20,7 @@
   locales = {}
 
   data_attributes ||= {}
-  data_attributes[:module] = 'gem-accordion'
+  data_attributes[:module] = 'govuk-accordion gem-c-accordion'
   data_attributes[:anchor_navigation] = anchor_navigation
```

Comment:
I think i've found an issue in Manuals Frontend in that the anchor links do not seem to expand the accordion sections that they are linked to. 

I think the issue is down to the naming convention used here `data_attributes[:module] = 'govuk-accordion gem-c-accordion'`. I think the module expects the format to be `gem-accordion`. As far as I can tell that's how it's initialising the component. 

```
      // eg selectable-table to SelectableTable
      function camelCaseAndCapitalise (string) { // gem-accordion
        return capitaliseFirstLetter(camelCase(string)) // GemAccordion
      }
```

Reply:
This makes sense, I updated from `gem-accordion` to `gem-c-accordion` during this last wave of changes to match convention but of course, needs  `gem-accordion`

I'm not a fan of this mismatch, it's confusing, let me see if I can align.  Thank @jon-kirwan